test/utf8/rlexe.xml

```diff
@@ -20,4 +20,10 @@
       <baseline />
     </default>
   </test>
+  <test>
+    <default>
+      <files>bugGH2386.js</files>
+      <baseline />
```

Comment:
I recall to check "pass" output we should omit "baseline" tag. "<baseline/>" means ignore output.